include/dali/core/geom/vec.h

```diff
@@ -79,6 +79,46 @@ template <typename T>
 struct is_scalar : std::integral_constant<bool, !is_mat<T>::value && !is_vec<T>::value> {};
```

Comment:
Can you add template variable aliases for those?

Reply:
Done.